tests/bullet/Demos/HelloWorld/parse.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
i'm not sure we should update external python files that have an upstream. in this file's case, we never use it, it's just part of bullet and we included it all. so i guess it would harmless to modify it, but it would mean we'd see a diff when comparing to upstream. again, in this case that's probably fine (I doubt we'll update bullet, or if we do, we'd do it as a whole, not incrementally).

so overall, I'm not sure what's best here.

Reply:
It seems bullet, poppler, and freetype haven't been updated from upstream since 2011 so personally I think it will be fine.

If you want I'll still happily exclude them on this PR.

Reply:
(I think eventually we will have to fix some of them, but probably only after we add CI settings for Python 3?)